benchopt/tests/test_benchmark_features.py

```diff
@@ -32,3 +39,18 @@ def test_template_solver():
     # Make sure that this error is not raised when listing all solvers from
     # the benchmark.
     DUMMY_BENCHMARK.get_solvers()
+
+
+def test_benchopt_min_version():
+    # Make sure that importing template_dataset raises an error.
+    with pytest.raises(RuntimeError, match="pip install -U"):
+        run([str(FUTURE_BENCHMARK_PATH)], 'benchopt', standalone_mode=False)
+
+    with CaptureRunOutput() as out:
```

Comment:
I don't understand why this is needed in this test, can you please explain to me?

Reply:
Ok got it